packages/mendel-middleware/index.js

```diff
@@ -29,6 +30,38 @@ function MendelMiddleware(opts) {
             return getPath({ bundle: bundle, hash: tree.hash });
         };
 
+        req.mendel.getExposed = function(variations) {
+            if (!variations) {
+                variations = 'base';
+            }
+
+            if (exposedBundleVariations[variations]) {
```

Comment:
Yeah I see the mistake. I assumed that `variations` would be string but it is an array. 

All your concerns are valid.